LBHackney-IT/lbh-fss-portal-frontend
# Feature/set password on onboarding

**Description**

On login, if a user needs to set a password they should be directed to **/account** page and a message should be displayed 

On login, if a user does not need to set a password they will be directed to **/organisation**  page and no message will be displayed on the **/account** page

**Test instructions**

- pull this branch (you will automatically be set as a user who needs to set their password) 
- first logout, then sign in (you can use any dummy email/password as long as they meet frontend validation) to check you are directed to **/account** page and can see a message asking you to set your password

-- 

- switch user to one that does not need to set their password, this can be done by:
    - navigating to **mock-api/account/GET.js** and commenting out "Tommie Dietrich" and uncommenting "Melody Zieme"
    - navigating to **mock-api/session/POST.js** and commenting out "Tommie Dietrich" and uncommenting "Melody Zieme"
- first logout, then sign in to check you are directed to **/organisation** page and when navigating to **/account** page you do not see a message asking you to set your password


**Ticket**
https://app.clickup.com/t/9e5u6x
